_src/assets/js/partials/localStorage.js

```diff
@@ -109,19 +118,34 @@ function printLocalStorage() {
     let valueTyp = saveData.typography - 1;
     let classTyp = typographiesValue[valueTyp];
 
-    function removeClassTyp() {
-      for (let i = 0; i < typographiesValue.length; i++) {
-        boxFont.classList.remove(typographiesValue[i]);
-      }
+
```

Comment:
In this point is the same, maybe a map function will be ok, if you consider that a good option